share/completions/cargo.fish

```diff
@@ -89,3 +89,4 @@ complete -c cargo -x -n '__fish_seen_subcommand_from update' -l precise -d 'Upda
 complete -c cargo -x -n '__fish_seen_subcommand_from yank' -l vers -d 'The version to yank or un-yank'
 complete -c cargo -n '__fish_seen_subcommand_from yank' -l undo -d 'Undo a yank, putting a version back into the index'
 
+set -e __fish_cargo_subcommands
```

Comment:
Please just set this as local instead of erasing it explicitly. That way it will shadow any (theoretical) global of that name.